python/runtime/pai/submitter.py

```diff
@@ -433,10 +452,21 @@ def submit_pai_train(datasource, estimator_string, select, validation_select,
                     train_table, val_table, path_to_save, params)
 
     # submit pai task to execute the training
-    cmd = get_pai_tf_cmd(conf, "file://" + path.join(cwd, JOB_ARCHIVE_FILE),
-                         "file://" + path.join(cwd, PARAMS_FILE), ENTRY_FILE,
-                         model_name, path_to_save, train_table, val_table, "",
-                         project)
+    if estimator_string.lower().startswith("randomforests"):
```

Comment:
I think we should equal to `randomforests` or `kemans` exactly,  instead of starting with.